src/app/modules/main/communities/minting/models/token_item.nim

```diff
@@ -0,0 +1,92 @@
+import strformat
+
+type
+  TokenType* {.pure.} = enum
+    #ERC20,
```

Comment:
Needed?

Reply:
Hmm, this may look a bit premature.
We start with NFTs, but we will have ERC20 in the end too.
However, maybe we will have a separated model for them, and I should remove token type for now.